src/nyc_trees/gulpfile.js

```diff
@@ -153,6 +153,7 @@ function browserifyTask(bundler) {
         })
         .bundle()
         .on('error', gutil.log.bind(gutil, 'Browserify Error'))
+        .on('error', process.exit.bind(process, 1))
```

Comment:
@kdeloach I tried to do something like that, looking for "watch" in ``process.argv``. Crashing is easy, but I was having problems getting watch to resume properly.